www/front_src/src/Resources/Graph/Performance/AnomalyDetection/AnomalyDetectionSlider.tsx

```diff
@@ -0,0 +1,316 @@
+import Checkbox from '@mui/material/Checkbox';
+import Button from '@mui/material/Button';
+
+import { IconButton, useRequest, putData } from '@centreon/ui';
+
+import {
+  labelMenageEnvelope,
+  labelMenageEnvelopeSubTitle,
+  labelCancel,
+  labelSave,
+  labelUseDefaultValue,
+} from '../../../translatedLabels';
+import { ResourceDetails, Sensitivity } from '../../../Details/models';
+
+import { countedRedCirclesAtom } from './anomalyDetectionAtom';
+import { CustomFactorsData } from './models';
+
+const useStyles = makeStyles((theme) => ({
+  body: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  bodyContainer: {
+    alignItems: 'center',
+    display: 'flex',
+    marginBottom: theme.spacing(2),
+    marginTop: theme.spacing(5),
+  },
+  confirmButton: {
+    marginLeft: theme.spacing(2),
+  },
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-evenly',
+    padding: theme.spacing(2),
+  },
+  footer: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+  },
+  header: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  icon: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  slider: {
+    '& .MuiSlider-mark': {
+      borderLeft: '1px solid',
+      height: theme.spacing(2),
+      width: 0,
+    },
+    '& .MuiSlider-thumb': {
+      height: theme.spacing(3),
+      width: 1,
+    },
+    '& .MuiSlider-valueLabel': {
+      backgroundColor: theme.palette.primary.main,
+      borderRadius: '50%',
+    },
+    '& .MuiSlider-valueLabel:before': {
+      width: 0,
+    },
+    '& .MuiSlider-valueLabelOpen': {
+      transform: 'translateY(-60%) scale(1)',
+    },
+    display: 'flex',
+    justifyContent: 'space-evenly',
+    width: theme.spacing(35),
+  },
+}));
+
+interface Props {
+  details: ResourceDetails;
+  isEnvelopeResizingCanceled?: boolean;
+  isResizeEnvelope?: boolean;
+  openModalConfirmation?: (value: boolean) => void;
+  sendFactors: (data: CustomFactorsData) => void;
+  sendReloadGraphPerformance: (value: boolean) => void;
+  sensitivity: Sensitivity;
+  setIsResizeEnvelope?: Dispatch<SetStateAction<boolean>>;
+}
+
+const AnomalyDetectionSlider = ({
+  sendFactors,
+  sensitivity,
+  details,
+  openModalConfirmation,
+  isEnvelopeResizingCanceled,
+  isResizeEnvelope,
+  sendReloadGraphPerformance,
+  setIsResizeEnvelope,
+}: Props): JSX.Element => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const [currentValue, setCurrentValue] = useState(sensitivity.current_value);
+  const [isDefaultValue, setIsDefaultValue] = useState(false);
+  const [isResizing, setIsResizing] = useState(false);
+  const [openTooltip, setOpenTooltip] = useState(false);
+  const { sendRequest } = useRequest({
+    request: putData,
+  });
+  const [countedRedCircles, setCountedRedCircles] = useAtom(
+    countedRedCirclesAtom,
+  );
+
+  const msgTooltip = `${countedRedCircles} points out of envelope`;
```

Comment:
```suggestion
  const tooltipMessage = `${countedRedCircles} points out of envelope`;
```
"points out of envelope" should be translated

Reply:
realized in the new pr